execution_engine/src/core/engine_state/mod.rs

```diff
@@ -2476,7 +2476,7 @@ where
         &self,
         correlation_id: CorrelationId,
         step_request: StepRequest,
-    ) -> Result<StepResult, Error> {
+    ) -> Result<StepResult, GetEraValidatorsError> {
```

Comment:
@EdHastingsCasperLabs It ended up being easiest to have this become a `GetEraValidatorsError`, but it's a bit odd and I'm not sure if there's a better way here.